### DISCUSSION TITLE
Cannot create random links

### DISCUSSION THREAD

**Discussion by FrankV1971** [YOURLS/YOURLS#3438] [Q&A] [answered]
We upgraded to Yourls 1.9.1 a couple of months ago. Today we noticed a problem with generating random links. The error below is shown in the PHP error log. PHP is version 8.1.11 (we upgraded from 8.0.3 that gave the same error today). When manually entering a Custom short URL works. (that is the reason we did not detect the problem till today. 

```
[12-Oct-2022 15:25:01 UTC] PHP Fatal error:  Uncaught BadMethodCallException: Class 'YOURLS\Database\YDB' does not have a method 'get_results' in C:\inetpub\shorturl\includes\vendor\aura\sql\src\AbstractExtendedPdo.php:113
Stack trace:
#0 C:\inetpub\shorturl\user\plugins\url-reuser\plugin.php(16): Aura\Sql\AbstractExtendedPdo->__call()
#1 C:\inetpub\shorturl\includes\functions-plugins.php(221): bammerch_urlreuser_reuse_id()
#2 C:\inetpub\shorturl\includes\functions.php(50): yourls_apply_filter()
#3 C:\inetpub\shorturl\includes\functions-shorturls.php(121): yourls_get_next_decimal()
#4 C:\inetpub\shorturl\admin\admin-ajax.php(21): yourls_add_new_link()
#5 {main}
  thrown in C:\inetpub\shorturl\includes\vendor\aura\sql\src\AbstractExtendedPdo.php on line 113
```

I checked here and in Google but could not find a solution. Anybody an idea what is going wrong?

Any help appreciated.

**Comment by dgw** [comment, marked as answer]
It is probably related to that url-reuser plugin I can't find anything about either through GH search or Google.

**Reply by FrankV1971** [reply]
Thnx that did it.
